shard-operator-parent/shard-operator-v2/src/main/resources/application.properties

```diff
@@ -1,3 +1,2 @@
 # Configuration of polling and timeout operations for resources relating to a ManagedBridge instance
-event-bridge.managed-bridge.poll.interval-milliseconds=5000
-event-bridge.managed-bridge.deployment.timeout-seconds=60
\ No newline at end of file
+event-bridge.managed-bridge.poll.interval-milliseconds=5000
```

Comment:
You can remove this if "time-out" is to be moved to the `manager`.

Reply:
sure, As team decide on to remove retry interval. I will update it accordingly.

Reply:
@manstis @vaibhavjainwiz Let's leave it in for now to reduce the number of changes in this PR. We have https://issues.redhat.com/browse/MGDOBR-1308 to discuss whether moving timeout back to the Manager is the right thing to do. Once we've had that discussion, we can make another (smaller PR) to update.

Reply:
@manstis Jira for same already exists.
https://issues.redhat.com/browse/MGDOBR-1308